kubernetes_topology/conf.yaml.example

```diff
@@ -0,0 +1,19 @@
+init_config:
+  #    tags:
+  #      - optional_tag1
+  #      - optional_tag2
+
+instances:
+  # The kubernetes topology check retrieves topology from the kubernetes api.
+  # By default we will assume we're running under docker and will use the address
+  # of the default router to reach the kubernetes api.
+  #
+  # To override, e.g. in the case of a standalone instance, use the following:
+  #
```

Comment:
use the following: <nothing>?
Remove or add the missing setting?

Reply:
made wording clearer